bin/node/runtime/Cargo.toml

```diff
@@ -154,6 +154,7 @@ std = [
 	"pallet-society/std",
 	"pallet-recovery/std",
 	"pallet-vesting/std",
+	"frame-try-runtime/std",
```

Comment:
I also added std here in order to be able to compile the crate with try-runtime and std.
(`cargo check -p node-runtime --features try-runtime`)

Otherwise it fails.

Reply:
The problem with that is that Cargo currently doesn't support optional dependencies and features. 

Not sure how this works :shrug:

Reply:
AFAICT this will pull frame-try-runtime crate everytime std is activated, which is not what we want, but is not bothering too much either.

Reply:
I think given the same situation with benchmark, let's move on and let it be, but a refactor issue for it will be appreciated. Maybe we can do it nicer someday.

Reply:
yes I think there is plan to support it with something like `"frame-try-runtime?/std"`

Ok then I'll merge and open an issue